609 keV ROIs

jnacar opened the pull request:
I imported Bi_214_g_E and created an exception for the 609 Bi204 peak that follows the similar behavior as the exception for Cs134.

bplimley commented on ROI_Maker.py:
```diff
@@ -48,9 +50,18 @@ def ROI_Maker(spectrum, energy, sub_regions='auto'):
-        bi_right_ch = (bi_right_peak - fwhm_channel,
-                       bi_right_peak + fwhm_channel)
+        bi_right_ch = (bi_right_peak + fwhm_channel,
+                       bi_right_peak - fwhm_channel)
```
As above, the `- fwhm_channel` should come before the `+`. This matters because `Gamma_Analysis` lines 197-198 use these values as indices for slicing an array, and you have to slice left to right. So flip these back, and also below for `cs_left_ch`. Thanks.

bplimley commented on ROI_Maker.py:
```diff
@@ -48,9 +50,18 @@ def ROI_Maker(spectrum, energy, sub_regions='auto'):
+        cs_fwhm = 0.05 * (604.72)**0.5
+        cs_fwhm_channel = int(region_size * (cs_fwhm - E0) / Eslope)
+        cs_peak_channel = int((604.72 - E0) / Eslope)
+        cs_left_peak = cs_peak_channel + compton_distance * cs_fwhm_channel
```
This should be `cs_peak_channel - compton_distance * cs_fwhm_channel` because we want a Compton region left of the 604 peak.

jnacar commented:
I forgot to put my comments when I did the commit. Sorry, still getting use to it. But here are my comments:

1) Changed bi_right_ch to (bi_right_peak - fwhm_channel, bi_right_peak + fwhm_channel)
2) Changed cs_left_ch to (cs_left_peak - fwhm_channel, cs_left_peak + fwhm_channel)
3) Changed cs_left_peak to cs_peak_channel - compton_distance * cs_fwhm_channel
